myExpenses/src/main/res/values-pt/aosp.xml

```diff
@@ -10,7 +10,6 @@
     <string name="default_action">"Ação padrão"</string>
     <string name="full_name">"Nome"</string>
     <string name="postal_country">"Țară"</string>
-    <string name="postal_country">"País"</string>
```

Comment:
This by the way removes the wrong string. Problem has been fixed in master. Please merge, and remove this change

Reply:
Done